docs/src/rippledAPIs.md.ejs

```diff
@@ -0,0 +1,49 @@
+# rippled APIs
+
+ripple-lib relies on [rippled APIs](https://ripple.com/build/rippled-apis/) for all online functionality. With ripple-lib version 0.22.0 and higher, you can easily access rippled APIs through ripple-lib. Use the `request()`, `hasNextPage()`, and `requestNextPage()` methods:
+* Use `request()` to issue any `rippled` command, including `account_currencies`, `subscribe`, and `unsubscribe`. [Full list of API Methods](https://ripple.com/build/rippled-apis/#api-methods).
+* Use `hasNextPage()` to determine whether a response has more pages. This is true when the response includes a [`marker` field](https://ripple.com/build/rippled-apis/#markers-and-pagination).
+* Use `requestNextPage()` to request the next page of data.
+
+## Listening to streams
+
+The rippled server can push updates to your client when various events happen.
+
+When you subscribe to a stream, you must also listen to the relevant message type(s). The available types in the table below.
+
+Type | Description
+---- | -----------
+`ledgerClosed` | Sent by the `ledger` stream when the consensus process declares a new fully validated ledger. The message identifies the ledger and provides some information about its contents.
+`validationReceived` | Sent by the validations stream when the server receives validation messages, also called validation votes, from validators it trusts.
```

Comment:
This stream will include all validations received, not just those the rippled trusts.
https://github.com/ripple/ripple-dev-portal/pull/341